src/ProjectTemplates/Shared/Project.cs

```diff
@@ -132,9 +130,7 @@ internal async Task<ProcessResult> RunDotNetBuildAsync(IDictionary<string, strin
             // Avoid restoring as part of build or publish. These projects should have already restored as part of running dotnet new. Explicitly disabling restore
             // should avoid any global contention and we can execute a build or publish in a lock-free way
 
-            var razorSDKarg = string.IsNullOrEmpty(RazorSdkDirectoryRoot) ? string.Empty : $"/p:RazorSdkDirectoryRoot={RazorSdkDirectoryRoot}";
-
-            using var result = ProcessEx.Run(Output, TemplateOutputDir, DotNetMuxer.MuxerPathOrDefault(), $"build --no-restore -c Debug /bl {razorSDKarg} {additionalArgs}", packageOptions);
+            using var result = ProcessEx.Run(Output, TemplateOutputDir, DotNetMuxer.MuxerPathOrDefault(), $"build --no-restore -c Debug /bl {additionalArgs}", packageOptions);
```

Comment:
Why do we need the rid now here?

Reply:
Further up? WASM produces outputs in a RID based directory. There's a flag to turn it off, but does not work: https://github.com/dotnet/sdk/issues/12114